lib/tasks/reset.rake

```diff
@@ -0,0 +1,20 @@
+namespace :hyku do
+  desc 'reset work and collection data across all tenants'
+  task works_and_collections: [:environment] do
+    confirm('You are about to delete all works and collections across all accounts, this is not reversable!')
+    Account.find_each do |account|
```

Comment:
Do you think we should make a tenant argument optional instead of resetting all of them?

Reply:
task name implies the intention is to do this across all tenants. So I'll take back my above comment and approving this PR

Reply:
I did change it today shana =-)